cypress/e2e/webComponents.cy.ts

```diff
@@ -37,22 +37,22 @@ describe('Web Components', () => {
     const handlers = {
       show: event => {
         expect(event.detail.target.tagName).to.eq('MY-DIALOG')
-        expect(event.detail.composedPath()[0].tagName).to.eq('FANCY-BUTTON')
+        expect(event.detail.composedPath()[0].tagName).to.eq('SLOT')
       },
       hide: event => {
         expect(event.detail.target.tagName).to.eq('MY-DIALOG')
-        expect(event.detail.composedPath()[0].tagName).to.eq('FANCY-BUTTON')
+        expect(event.detail.composedPath()[0].tagName).to.eq('SLOT')
       },
     }
 
     cy.spy(handlers, 'show').as('show')
     cy.spy(handlers, 'hide').as('hide')
     cy.window().its('instance').invoke('on', 'show', handlers.show)
     cy.window().its('instance').invoke('on', 'hide', handlers.hide)
-    cy.get('my-dialog').shadow().find('fancy-button').first().click()
```

Comment:
The tests were passing because `.click()` was clicking the exact shadow root. A `.realClick()` however would click the inner `<slot>` and cause the test to fail, as reported in #712.